apps/design/frontend/src/export_screen.tsx

```diff
@@ -1,22 +1,58 @@
-import { H1, P, Button, MainContent, MainHeader } from '@votingworks/ui';
+import { useState } from 'react';
+import {
+  H1,
+  P,
+  Button,
+  MainContent,
+  MainHeader,
+  useQueryChangeListener,
+  LoadingButton,
+  Icons,
+} from '@votingworks/ui';
 import fileDownload from 'js-file-download';
 import { useParams } from 'react-router-dom';
 import { getDisplayElectionHash } from '@votingworks/types';
+import { assertDefined } from '@votingworks/basics';
 import {
   exportAllBallots,
   exportElectionPackage,
   exportTestDecks,
+  getElectionPackage,
 } from './api';
 import { ElectionNavScreen } from './nav_screen';
 import { ElectionIdParams } from './routes';
+import { downloadFile } from './utils';
 
-export function ExportScreen(): JSX.Element {
+export function ExportScreen(): JSX.Element | null {
   const { electionId } = useParams<ElectionIdParams>();
+
+  const electionPackageQuery = getElectionPackage.useQuery(electionId);
   const exportAllBallotsMutation = exportAllBallots.useMutation();
-  const exportTestDecksMutation = exportTestDecks.useMutation();
   const exportElectionPackageMutation = exportElectionPackage.useMutation();
+  const exportTestDecksMutation = exportTestDecks.useMutation();
+
+  const [exportError, setExportError] = useState<string | undefined>(undefined);
```

Comment:
Do we need state for this? I think we can just use `electionPackageQuery.data.task?.error`

Reply:
Yeah good question! I landed on a separate state value because:

1. I didn't want the error message to be displayed on first load of the page if the last task, however long ago it was, failed
2. I wanted clicking to export again to immediately dismiss the error, without having to wait for relevant React Query state to update
3. I wanted clicking any export button, not just the export election package button, to dismiss the error

Reply:
Makes sense!